content/tracking/1-setup.md

```diff
@@ -0,0 +1,169 @@
++++
+title = "Setup Your Tracking"
+weight = 1
+post = ""
++++
+
+To instrument tracking, you will need to install and configure the tracker package in your mobile app.
+
+#### Step 1: Install the tracker package
+
+{{< tabs groupId="platform" >}}
+
+{{% tab name="iOS" %}}
+
+You can install the tracker using SPM as follows:  
+1. In Xcode, select File > Swift Packages > Add Package Dependency.
+2. Add the URL where to download the library: https://github.com/snowplow/snowplow-objc-tracker
+
+To learn about other options for installing the tracker (e.g. using CocoaPods or Carthage), [see the mobile tracker documentation](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/mobile-trackers/installation-and-set-up/).
+
+{{% /tab %}}
+{{% tab name="Android" %}}
+
+The tracker can be installed using Gradle. Add the following to your `build.gradle` file:
+
+```
+dependencies {
+  ...
+  // Snowplow Android Tracker
+  implementation 'com.snowplowanalytics:snowplow-android-tracker:3.+'
+  // In case 'lifecycleAutotracking' is enabled
+  implementation 'androidx.lifecycle-extensions:2.2.+'
+  ...
+}
+```
+
+{{% /tab %}}
+{{% tab name="React Native" %}}
+
+To install the tracker, add it as a dependency to your React Native app:
+
+```bash
+npm install --save @snowplow/react-native-tracker
+```
+
+{{% /tab %}}
+{{% tab name="Flutter" %}}
+
+Add the Snowplow tracker as a dependency to your Flutter application:
+
+```bash
+flutter pub add snowplow_tracker
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Step 2: Import the tracker package
+
+We recommend creating a class or a file with helper functions (e.g., `Tracker.swift`) to manage all the Snowplow related code in your project.
```

Comment:
Is it worth moving this inside the tabs so you can change the example filename to be relevant for each language? Or remove the `e.g.` bit?